src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -83,11 +83,20 @@ public Xcode(string target, string arch)
             }
         }
 
+        bool hardenedRuntime = false;
+        bool hardenedRuntimeUseJit = false;
+        if (Target == TargetNames.MacCatalyst && !(forceInterpreter || forceAOT)) {
+            hardenedRuntime = true;
```

Comment:
this is probably not right since even without the JIT we probably still need the hardened runtime and the "disable library validation" entitlement to load unsigned dylibs.  I didn't check.

Reply:
I'm wrong.  for the interpreter we only need the ICU_DAT fix.  It otherwise works without the hardened runtime.